tests/BaseTestCase.php

```diff
@@ -94,18 +94,18 @@ public function testModelLoadingCacheInformation()
 
     public function testGetConnectionByTableName()
     {
-        $connectionBefore = Doctrine_Core::getConnectionByTableName('entity');
+        $connectionBefore = Doctrine_Core::getConnectionByTableName('tag');
```

Comment:
The `entity` table represents several different classes (User and Group) which I think ends up causing issues here. I'm not clear why it seems to only have issues in some environments (this test passed fine on Travis at the time).  It's been an issue on my local machine since I upgraded to PHP 7.4.  This fixes the test, but not the issue with this `getConnectionByTableName` method and inherited classes.